src/data/trace.ts

```diff
@@ -93,11 +97,29 @@ export interface AutomationTraceExtended extends AutomationTrace {
   error?: string;
 }
 
+export interface ScriptTrace extends BaseTrace {
+  domain: "script";
+}
+
+export interface ScriptTraceExtended extends ScriptTrace {
+  trace: Record<string, ActionTraceStep[]>;
+  context: Context;
+  config: ScriptConfig;
+  blueprint_inputs?: BlueprintScriptConfig;
+  error?: string;
```

Comment:
`trace` , `context`, `blueprint_inputs` and `error?` are the same for both extended.
Does it make sense to add a `BaseTraceExtended`  interface that both extend?

Reply:
Stupid TS doesnt support extend from multiple interfaces... but this could work:

```ts
export type AutomationTraceExtended = AutomationTrace &
  BaseTraceExtended & {
    config: ManualAutomationConfig;
    blueprint_inputs?: BlueprintAutomationConfig;
  };
```

Reply:
NVM, TS does support multiple extends